src/sentry.js

```diff
@@ -0,0 +1,31 @@
+const Sentry = require("@sentry/node");
+
+function initSentry({ inputs }) {
+  const config = {
+    dsn: "TBD",
```

Comment:
What's the best place to store DSN? Should it be a secret or not?

Reply:
You can use it here, np
It's not a secret